src/pages/settings/Wallet/ActivatePhysicalCardPage.tsx

```diff
@@ -110,8 +108,8 @@ function ActivatePhysicalCardPage({
             return;
         }
 
-        CardSettings.activatePhysicalExpensifyCard(lastFourDigits, cardID);
-    }, [lastFourDigits, cardID]);
+        CardSettings.activatePhysicalExpensifyCard(lastFourDigits, physicalCard?.cardID ?? 0);
+    }, [lastFourDigits, physicalCard?.cardID]);
```

Comment:
Maybe we should default to a negative ID

Reply:
We could but isn't 0 also a wrong card ID and would also cause `CardSettings.activatePhysicalExpensifyCard` to fail(which is what we want when physical card doesn't have cardID). Do we have some special negative number for such cases?

Reply:
Yeah 0 is an incorrect card ID. I think we've historically used -1 but I need to double check. But let me know if you'll try returning early instead